dockerfiles/build.include

```diff
@@ -0,0 +1,151 @@
+#!/bin/sh
```

Comment:
finally, I prefer to keep this file in this PR ( and prioritize another issue for this ) as this script is not production code and would not change a lot and is quite tiny